zilencer/management/commands/populate_db.py

```diff
@@ -32,7 +33,7 @@
 from six.moves import range
 
 settings.TORNADO_SERVER = None
-
+DEFAULT_NOTIFICATION_STREAM_NAME = "notifications"
```

Comment:
You should just use `Realm.DEFAULT_NOTIFICATION_STREAM_NAME` here -- no need to make up a new stream name.